README.adoc

```diff
@@ -144,58 +143,44 @@ Create the `InventoryManager` class in the
+Apply the `@Timed` annotation to the `get()` method to track how frequently the method is
 invoked and also how long it takes for the invocation to complete. This annotation has these
 metadata fields:
 
 |===
-|`name` | Optional and sets the name of the metric
-|`description` | Optional and describes the purpose of the metric annotation
+|`name` | Optional; Use this to name of the metric
+|`description` | Optional; Use this to describe the purpose of the metric
 |===
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of
-systems methods when you visit the `http://localhost:9080/inventory/systems` URL. This URL retrieves the information for a
-list of all previously registered hosts. See the additional metadata fields:
+Apply the `@Counted` annotation to the `list()` method to count how many times the
+`http://localhost:9080/inventory/systems` URL is accessed. Note the additional metadata field:
 
 |===
-| `absolute` | if `true`, it names the metric the name of the method. If `false`, it attaches the package name and class name
-before the method name
-| `monotonic` | if `true`, the counter counts the number of invocations of the annotated method. If `false`, the counter counts the
-number of currently executing invocations of the annotated method. In other words, it makes the counter increment
-when the annotated method is called and decrement when it returns.
+| `monotonic` | Set to `true` to count the total number of invocations of the annotated method i.e.
+the total number of accesses.
 |===
 
-The `@Gauge` annotation on the `getTotal()` method tracks the number of systems that are stored in
-the inventory. A system is stored in the inventory when you visit the
-`http://localhost:9080/inventory/systems/localhost` URL to retrieve the information for a specific host.
-See the additional metadata field:
+Apply the `@Gauge` annotation to the `getTotal()` method to track the number of systems that are stored in
+the inventory. Note that when the value of the gauge is retrieved, the underlying `getTotal()` method
+is called to return the size of the inventory.
 
-|===
-| `unit` | Sets the unit of the metric
-|===
-
-Note that, when the value of the gauge is retrieved, the underlying `getTotal()` annotated method is
-called to return the value.
-
-These annotations are registered by default to the `MetricRegistry` singleton object.
-This object is queried for metadata and metrics when you call the `https://localhost:9443/metrics/application`
-MicroProfile Metrics endpoint, which is provided for you. Also, the `@Counted`, `@Gauge`, and `@Timed` annotations can be
-used in services, such as the `inventory` service in the
-`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file.
+Additional information about these annotations, relevant metadata fields and more is available at
```

Comment:
Where is `unit` and `absolute` fields. I do not think we need to point out to other websites (they will have to look for it) we should provide that